app/controllers/api/users_controller.rb

```diff
@@ -62,8 +62,34 @@ def set_current_group_resource(_type, id, data)
       raise BadRequestError, "Cannot set current_group - #{err}"
     end
 
+    def revoke_sessions_collection(type, data)
+      revoke_sessions_resource(type, current_user.id, data)
+    end
+
+    def revoke_sessions_resource(type, id, _data)
```

Comment:
Should these 2 methods be private since they aren't routable?

Reply:
those method could be probably private but we don't have such methods set as private in other controllers, so let's leave as it is for now, maybe it is intentionally for some reason.